src/vnsw/agent/cmn/agent.cc

```diff
@@ -485,7 +488,7 @@ static bool interface_exist(string &name)
 		}
 		ifs++;
 	}
-	if_freenameindex(head);
+	osspecific_if_freenameindex(head);
```

Comment:
This function is not present on Linux I believe. This means it would break compatibility.